docs/self-managed/installations/docker-compose.md

```diff
@@ -175,6 +175,13 @@ sed -i."" "s/REPLACE_ME_BY_JOBS_REDIS_PASSWORD/$(openssl rand -hex 16)/g" .env
 sed -i."" "s/REPLACE_ME_BY_S3_SECRET_KEY/$(openssl rand -hex 16)/g" .env
 ```
 
+### 📋 (Optional) Add your custom CA
+
+If your GitLab instance is using a TLS certificate signed with your own
+Certificate authority (CA):
+
+1. Add the CA certificate file in `.docker/ca-certificates`
```

Comment:
It's weird that we're starting a list with 1. but in fact we only have one entry.

Also why are we talking about custom certificate in this page, shouldn't be specific to the docker compose custom certs page ?

Reply:
That's two different cases: 
- Compose custom certificate allows running R2Devops with certificate generated by the client (and not get it automatically using let's encrypt)
- The CA certificate option allows adding a new Certificate Authority in R2Devops to allow it to recognize certificates signed by the client Certificate Authority